tools/pd-ctl/pdctl/command/global.go

```diff
@@ -69,7 +69,7 @@ func InitHTTPSClient(pdAddrs, caPath, certPath, keyPath string) error {
 			&http.Transport{TLSClientConfig: tlsConfig}, pdControlCallerID),
```

Comment:
Should we clone here also to avoid the same problem appearing somewhere later?

Reply:
In fact, our pd-ctl doesn't support concurrency, so I think that's probably not necessary.